src/lib/icons/zoom_in.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="10" height="10" viewBox="0 0 10 10">
+    <g fill="none" fill-rule="evenodd">
```

Comment:
Is this wrapping tag needed?

Reply:
Yeah looks like a little black box appears without it